src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -222,21 +262,10 @@ public Uri CreateProjectDashboardUrl(ConnectionInformation connectionInformation
 
         internal /*for testing purposes*/ static string CreateQualityProfileExportUrl(QualityProfile profile, string language, string exporter)
         {
+            // TODO: why name and not key? why language is needed at all, profiles are per language
```

Comment:
Sent an email about it, Matthew was copied, in case we'll get a reply when I'm not around and we need to do something.